src/modules/deal-investor/dto/create-deal-investor-request.dto.ts

```diff
@@ -0,0 +1,67 @@
+import { ValidatedDateOnlyApiProperty } from '@ukef/decorators/validated-date-only-api-property.decorator';
+import { ValidatedStringApiProperty } from '@ukef/decorators/validated-string-api-property.decorator';
+import { DateString, UkefId } from '@ukef/helpers';
+
+export type CreateDealInvestorRequest = CreateDealInvestorRequestItem[];
+
+export class CreateDealInvestorRequestItem {
+  @ValidatedStringApiProperty({
+    description: 'The unique identifier of the portfolio associated to the business entity. The UKEF portfolio is E1.',
+    example: 'E1',
+    minLength: 0,
+    maxLength: 2,
+    required: false,
+    default: 'E1',
+  })
+  readonly portfolioIdentifier: string;
+
+  @ValidatedStringApiProperty({
+    description: "The unique identifier for the deal within the portfolio. It will be a 10-digit code beginning with either '0020', '0030', or '0040'.",
+    example: '0020900111',
+    length: 10, // TO-DO: check
+    pattern: /^00[2-4]0\d{6}$/,
+  })
+  readonly dealIdentifier: UkefId;
+
+  @ValidatedStringApiProperty({
+    description: 'The lender type code for the investor party of the deal.',
+    example: '500',
+    minLength: 0,
+    maxLength: 3,
+    required: false,
+    default: '500',
+  })
+  readonly lenderType?: string; // TO-DO: should this be numeric?
+
+  @ValidatedDateOnlyApiProperty({
+    description: 'The effective date on the deal record in YYYY-MM-DD format.',
```

Comment:
Good point. In light of this and the comments below about using 'deal investor record' instead of 'deal record', I've changed it to `"The effective date on the deal investor record. If the date provided is in the future, it will be replaced by today's date."`